ecl/eclagent/eclagent.cpp

```diff
@@ -2150,6 +2141,34 @@ void EclAgent::runProcess(IEclProcess *process)
         }
     }
 
+    // a component may specify an alternate name for the agent/workflow memory area,
+    // e.g. Thor specifies in "eclAgentMemory"
+    const char *jobMemorySectionName = agentTopology->queryProp("@jobMemorySectionName");
+    if (isEmptyString(jobMemorySectionName))
+        jobMemorySectionName = "jobMemory";
+    //Get memory limit. Workunit specified value takes precedence over config
+    unsigned memLimitMB = queryWorkUnit()->getDebugValueInt("hthorMemoryLimit", 0);
+    if (0 == memLimitMB)
+    {
+        if (isContainerized())
+        {
+            // not being picked up in Thor..
+            // because it's under eclAgentResources ? (but is that generated in config?)
```

Comment:
Should answer this question

Reply:
should have been removed.
This was solved via template foo in this PR, thor.yaml ensures that "resources" is available as a copy of eclAgentResources.